docsteady/spec.py

```diff
@@ -168,6 +168,32 @@ def process_steps(self, test_script):
         return teststeps_sorted
 
 
+def get_lvv_details(key):
+    """
+    get LVV information from Jira
+    :param key: LVV jira Key
+    :return: needed parameter (first stage, only high level req, if available in Jira
+    """
+    lvv = dict()
+    lvv['high_level_req'] = []
+    if key != "":
+        resp = requests.get(
+            Config.ISSUE_URL.format(issue=key),
+            auth=Config.AUTH
+        )
+        if resp.status_code != 200:
+            print("Unable to download")
+            print(resp.text)
+            sys.exit(1)
+        lvv_resp = resp.json()
+        if lvv_resp['fields']['customfield_13515']:
+            raw0 = lvv_resp['fields']['customfield_13515'].split(', [')
```

Comment:
I think this might be clearer with a regexp.  I'm not sure exactly what the customfield contents are, but something like this might work (with the following for loop removed):
```suggestion
            lvv['high_level_req'] = re.findall(r'\[(.+?)\|', lvv_resp['fields']['customfield_13515'])
```